### ISSUE TITLE
Codon whl is not a supported wheel on this platform

### ISSUE THREAD

**Issue by dawnmy** [exaloop/codon#68]
Thank you for the new release. I was trying to install Codon package in Python, but it did not work on both MacOS 13 (with Intel CPU) and AlmaLinux 8.5 (AMD EPYC 7352). I got similar error:
on macOS:
```
ERROR: codon-0.15.1-cp39-cp39-macosx_10_16_x86_64.whl is not a supported wheel on this platform
```
on Linux:
```
ERROR: codon-0.15.1-cp36-cp36m-manylinux2014_x86_64.whl is not a supported wheel on this platform
```
The error seems to be because of a new version Python was used. Could you also make the package available for Python3.8-3.11?

Thank you!

**Comment by dawnmy** [comment]
On macOS, I could install it with Python3.9. But on Linux, it did not work even after switched to Python 3.6

**Comment by inumanag** [comment]
Yes, Cython is very pedantic about the versions. We'll provide the binaries for other Python versions soon. In the meantime, can you please try installing from the source (`cd codon && CODON_DIR=<path_to_codon> pip install extra/python`)?

**Comment by dawnmy** [comment]
Thank you for the suggestion. I just tried to install it from source, but got an error: 
```
No module named 'config.config'
```
After installed the package `config` with `pip install config`, the error persists.

**Comment by layerkugou** [comment]
good

**Comment by inumanag** [comment]
Fixed in #119
